data.sql

```diff
@@ -5,4 +5,14 @@ VALUES
 	(1,'Agumon', TO_DATE('03/02/2020', 'MM/DD/YYYY'), 0, true, 10.23),
 	(2,'Gabumon', TO_DATE('11/15/2020', 'MM/DD/YYYY'), 2, true, 8),
 	(3,'Pikachu', TO_DATE('01/07/2021', 'MM/DD/YYYY'), 1, false, 15.04),
-	(4,'Devimon', TO_DATE('05/12/2017', 'MM/DD/YYYY'), 5, true, 11);
\ No newline at end of file
+	(4,'Devimon', TO_DATE('05/12/2017', 'MM/DD/YYYY'), 5, true, 11);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+		(5,'Charmander', TO_DATE('02/08/2020', 'MM/DD/YYYY'), 0, false, -11),
```

Comment:
- Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'-11.0'`

Reply:
@zieeco if you pay attention closely we do have integer numbers as well in the same column. I am wondering how to assign different identifiers for the same column. would you please guide me?